src/Castle.Windsor.Tests/Diagnostics/DuplicatedDependenciesDiagnosticTestCase.cs

```diff
@@ -31,9 +31,6 @@ public class DuplicatedDependenciesDiagnosticTestCase : AbstractContainerTestCas
 		protected override void AfterContainerCreated()
 		{
 			var host = Kernel.GetSubSystem(SubSystemConstants.DiagnosticsKey) as IDiagnosticsHost;
-#if SILVERLIGHT
-			host.AddDiagnostic<IDuplicatedDependenciesDiagnostic>(new DuplicatedDependenciesDiagnostic(Kernel));
-#endif
```

Comment:
Not sure I understand what needs to be done here. Add back this line that was under `#if SILVERLIGHT` ?